azure-functions/src/bindings/twilio_sms_message.rs

```diff
@@ -0,0 +1,108 @@
+use crate::{
+    rpc::{typed_data::Data, TypedData},
+    FromVec,
+};
+use serde_derive::{Deserialize, Serialize};
+use serde_json::{to_string, to_value, Value};
+
+/// Represents the Twilio SMS message output binding.
+///
+/// The following binding attributes are supported:
+///
+/// | Name          | Description                                                                                                                                        |
+/// |---------------|----------------------------------------------------------------------------------------------------------------------------------------------------|
+/// | `name`        | The name of the parameter being bound.                                                                                                             |
+/// | `account_sid` | The name of an app setting that holds your Twilio Account SID. If not set, the default app setting name is "AzureWebJobsTwilioAccountSid".         |
+/// | `auth_token`  | The name of an app setting that holds your Twilio authentication token. If not set, the default app setting name is "AzureWebJobsTwilioAuthToken". |
+/// | `from`        | The default phone number that the SMS text is sent from.                                                                                           |
+/// | `body`        | The default SMS message body to use.                                                                                                               |
+///
+/// # Examples
+///
+/// An example HTTP-triggered function that outputs a Twilio SMS message:
+///
+/// ```rust
+/// use azure_functions::{
+///     bindings::{HttpRequest, TwilioSmsMessage},
+///     func,
+/// };
+/// use std::borrow::ToOwned;
+///
+/// #[func]
+/// #[binding(name = "$return", from = "+15555555555")]
+/// pub fn send_sms(req: HttpRequest) -> TwilioSmsMessage {
+///     let params = req.query_params();
+///
+///     TwilioSmsMessage {
+///         to: params.get("to").unwrap().to_owned(),
+///         body: params.get("body").map(ToOwned::to_owned),
+///         ..Default::default()
+///     }
+/// }
+/// ```
+#[derive(Debug, Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
Considering the attributes of the struct are all one word, seems unnecessary but :man_shrugging:

Reply:
I tend to leave it for any future fields :smile: